templates/blog/post_list.html

```diff
@@ -0,0 +1,18 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Post's List:</h1>
+  <ul>
+    {% for post in post_list%}
+      <li><a href="{% url 'blog:post-detail' post.id %}">{{ post.title }}</a>
```

Comment:
The URL tag 'blog:post-detail' might be incorrect if the URL pattern is not defined as such in your Django app. Ensure that the URL pattern name matches exactly with what is defined in your 'urls.py'.